checkClock/test/controllers/login_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class LoginControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
If this code are not needed please delete it

Reply:
Done